### PR TITLE
Timeline - show all nested data when tracks are collapsed

### PR DESCRIPTION
**Pull request by adamabeshouse**
Fixes https://github.com/cBioPortal/cbioportal/issues/7887

Uncollapsed:
image

Collapsed:
image

[Link](https://cbioportal.mskcc.org/patient?caseId=P-0000012&studyId=mskimpact_test_june)

### CODE REVIEW

**Comment by jjgao** [comment]
Cool! I am wondering if we can try to collapse everything into one line? It would be useful to display a line in Tumor Evolution tab. And it would also enable cohort level visualization (one line per patient).

**Comment by adamabeshouse** [comment]
@jjgao we could definitely collapse everything into one line. I will mock up an example using the above data

**Comment by adamabeshouse** [comment]
@jjgao needs some polishing but heres the rough idea. is it promising?
image

**Comment by jjgao** [comment]
Thanks, @adamabeshouse. I think it's promising. We will need different colors for different sections. Also maybe also good to have the top lines on top. Should we design it in a separate PR?

**Comment by adamabeshouse** [comment]
@jjgao yes, I agree it should be a different PR, this is going to be quite involved with a lot of UI decisions

**Comment by adamabeshouse** [comment]
@alisman @jjgao should we merge this as-is and work on refinements later?

**Review by jjgao — APPROVED** [review]

**Comment by jjgao** [comment]
@adamabeshouse sounds good. Thanks!

File: packages/cbioportal-clinical-timeline/src/TimelineTracks.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { flattenTracks, sortNestedTracks } from './lib/helpers';
 import CustomTrack, { CustomTrackSpecification } from './CustomTrack';
 import { TICK_AXIS_HEIGHT } from './TickAxis';
 import { useObserver } from 'mobx-react-lite';
+import { ToolTipModel } from '../../../src/pages/studyView/charts/barChart/BarChartToolTip';
```
Comment (alisman) [inline packages/cbioportal-clinical-timeline/src/TimelineTracks.tsx:12]: i'm pretty sure this is the problem.  i have a branch building to test.   1) this import is not used. 2) packages shouldn't import from main project

**Review by alisman — APPROVED** [review]

**Pull request merged**